_activities/idds.md

```diff
@@ -0,0 +1,25 @@
+---
+title: intelligent Data Delivery Service
+author: Wen Guan
+layout: plain
+---
+
+## Introduction
+
+iDDS(intelligent Data Delivery Service) is an intelligent Data Delivery Service. It's an ATLAS/IRIS-HEP joint project. It’s designed to intelligently transform and deliver the needed data to the processing workflow in a fine-grained approach for High Energy Physics workloads. It will not only reduce the need for replicas, but also enable benefits such as decreasing the time period of caching transient data, transforming expensive replicas to cheaper format data at remote sites and only cache cheaper new data for processing. iDDS will also work to orchestrate the WorkFlow Management System (WFMS) and the Distributed Data Management (DDM) systems to trigger them to process the data as soon as possible. In addition, iDDS will have intelligent algorithms to adjust the lifetime of cache, the format transformation and the delivery destination. The iDDS will increase the efficiency of data usage, reduce storage usage for processing and speed up the processing workflow.
+
+## Main description
+
+The iDDS is designed as a standalone experiment agnostic service. It's implemented based on a Work/Workflow structure, where a Work is a transformation/processings and a Workflow is a bunch of Works and their relationship, such as DAG. New use cases can be imported to iDDS by implementing a new Work with some hook functions.
```

Comment:
It's -> It is

a Work -> Work

bunch of -> collection of ("bunch" is rather informal)

"such as a DAG" - such as or always as a DAG? I'm trying to imagine what wouldn't be a DAG.

a new Work -> a new Work Item (it's a bit awkward as "work" is normally an uncountable noun in English, so to use it as the name for countable items grates a bit).